moduleroot/.gitlab-ci.yml.erb

```diff
@@ -91,6 +91,22 @@ before_script:
   allow_failure: true
 <%         end -%>
 <%       end -%>
+<%       if options['except'] -%>
```

Comment:
This could be merged into a single `if` instead of nested e.g.

```
<% unless options['except'].nil? || options['except']['check'].nil? -%>
    except:
<% options['except'][check].each do |ref| -%>
   - <%= ref %>
<%  end -%>
<%  end -%>
```